### PR TITLE
feat(dlp): added a sample for dlp_deidentify_table_with_multiple_crypto_hash

### PR DESCRIPTION
**Pull request by AarshD-crest**
## Description

- Added a sample for `dlp_deidentify_table_with_multiple_crypto_hash` with a unit test case. 

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [X] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [X] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [X] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [X] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by telpirion — APPROVED** [review]

File: dlp/snippets/deid/deid_test.go [hunk 1]
```diff
@@ -570,3 +570,22 @@ func TestReidTextDataWithFPE(t *testing.T) {
...
+	var buf bytes.Buffer
+
+	if err := deIdentifyTableWithMultipleCryptoHash(&buf, tc.ProjectID, "your-transient-crypto-key-1", "your-transient-crypto-key-2"); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "Table after de-identification :"; !strings.Contains(got, want) {
+		t.Errorf("TestDeIdentifyTableWithMultipleCryptoHash got %q, want %q", got, want)
+	}
+	if want := `{string_value:\"my email is user1@example.org and phone is 858-555-0222\"}`; strings.Contains(got, want) {
```
Comment (annadata) [inline dlp/snippets/deid/deid_test.go:585]: Please assert to confirm both info types have been de-identified.

File: dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// deIdentifyTableWithMultipleCryptoHash Transforms findings using two separate cryptographic hash transformations.
+func deIdentifyTableWithMultipleCryptoHash(w io.Writer, projectID, transientKey1, transientKey2 string) error {
```
Comment (annadata) [inline dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go:27]: please use transientKeyName1, transientKeyName2

File: dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+	// Define type of de-identification as cryptographic hash transformation.
+
+	// Specify the transformation for transientKey1
+	primitiveTransformation1 := &dlppb.PrimitiveTransformation{
+		Transformation: &dlppb.PrimitiveTransformation_CryptoHashConfig{
+			CryptoHashConfig: cryptoHashConfig1,
+		},
+	}
+
+	// Specify the transformation for transientKey1
```
Comment (annadata) [inline dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go:130]: transientKey2

**Review by annadata — APPROVED** [review]

File: dlp/snippets/deid/deid_test.go (hunk 1 above)
Comment (AarshD-crest) [reply dlp/snippets/deid/deid_test.go]: Updated.

File: dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go (hunk 2 above)
Comment (AarshD-crest) [reply dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go]: Updated it.

File: dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go (hunk 3 above)
Comment (AarshD-crest) [reply dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go]: updated it.

**Pull request merged**